css-two-column-layout/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main class="container">
+    <div class="row center flow-column banner center-column">
+      <h1 class="karla-font" id="header-title">SCALE, SPEED, AGILITY</h1>
+      <p id="header-summary" class="center-text small-text karla-font">
+        We've dedicated thousands of man hours to making
+        healthcare institutions in the Helium network run
+        10x more efficiently and deliver an unparalleled
+        quality of care.
+      </p>
+    </div>
+
+    <section id="feature-section">
+      <div class="row center">
+        <div class="col-one-sixth">
+          <div class="image-wrapper remove-image-distance">
+            <img src="./images/software.svg" alt="Software">
+          </div>
+        </div>
+        <div class="col-one-third">
+          <h2 class="feature-header big-text karla-font">INTUITIVE,<br> SMART SOFTWARE</h2>
+          <p class="small-text karla-font set-paragraph-width">
+            We build products that transform how healthcare
+            facilities run across Africa - making them radically
+            more efficient operationally and finacnially, a complete
+            turnaround.
+          </p>
+        </div>
+        <div class="col-one-sixth">
+          <div class="image-wrapper remove-image-distance">
+            <img src="./images/practice.svg" alt="Practice">
+          </div>
+        </div>
+        <div class="col-one-third increase-padding-right">
+          <h2 class="feature-header big-text karla-font">PRACTICE MEDICINE,
+              NOT ADMINISTRATION
+          </h2>
+          <p class="small-text karla-font set-paragraph-width">
+            Our all-in-one electronic health records (EHR), hospital
+            management system (HMIS) &amp; billing system helps streamline
+            your facility, reduce waste and improve quality of care.
+          </p>
+
+        </div>
```

Comment:
With this organization, this looks like the two sets of images and text are related to each other. It would be best if you organized this into chunks of `column-half`s, with each `column-half` representing one set of image and text, then make a `row` inside that `column-half` to control the image and text.

Reply:
Sounds good, I'll make the changes today.